Filters/exceptions.txt

```diff
@@ -1,3 +1,6 @@
+! TSB Bank mobile banking app
+! Blocked by CNAME online-metrix.net
+@@||check2.tsb.co.uk^|
```

Comment:
Should we leave that number `2`? I don't know if they have other similar domains, but what about `@@||check*.tsb.co.uk^|`?

Reply:
Try nslookup. They use check2 only

Reply:
Ok.